src/game/Tactical/ShopKeeper_Interface.cc

```diff
@@ -1896,10 +1908,10 @@ static void DisplayArmsDealerCurrentInventoryPage(void)
 					fDisplayHatchOnItem = TRUE;
 				}
 				//if the dealer repairs
-				else if( ArmsDealerInfo[ gbSelectedArmsDealerID ].ubTypeOfArmsDealer == ARMS_DEALER_REPAIRS )
+				else if (DoesDealerDoRepairs(gbSelectedArmsDealerID))
 				{
 					//if the item is damaged
-					if( ArmsDealerInfo[ gbSelectedArmsDealerID ].uiFlags & ARMS_INV_ITEM_NOT_REPAIRED_YET )
+					if (gpTempDealersInventory[usCnt].uiFlags & ARMS_INV_ITEM_NOT_REPAIRED_YET)
```

Comment:
This has to be a bug. The original code was looking for a `ARMS_INV_ITEM` flag in the `ARMS_DEALER_INFO`.

Reply:
However the effect was that it was always false. Shouldn't this check be negated? As in, only hatch items that have been repaired already?

Reply:
Indeed... the hatch seems totally useless. 
 1. The repaired item are placed in the bottom section
 2. The repair ETA cannot be read at all with hatch  

See the below screenshot:
![image](https://user-images.githubusercontent.com/63151803/90509655-40f9b880-e18c-11ea-9a19-768f443aa6a8.png)

I think we should just remove the check and leave `fDisplayHatchOnItem` as FALSE.

Reply:
Yes, let's remove — the red text is enough.